Localization/zh-cn.cfg

```diff
@@ -924,7 +922,7 @@ Localization
 
         #MechJeb_Utilities_label1 = 自动分级 <<1>> 启动
         #MechJeb_Utilities_label1_1 = 一次
-        #MechJeb_Utilities_label2 = 飞机安全裕度
+        #MechJeb_Utilities_label2 = 飞机安全速度
```

Comment:
`safety margin` is not the `safety speed`. Come on, you should see the `en-us.cfg` first.